packages/venia-concept/src/components/Input/input.css

```diff
@@ -4,7 +4,7 @@
 }
 
 .requiredSymbol {
-    background-color: rgb(var(--venia-text-spot));
+    background-color: black;
```

Comment:
This does fix the color, however there looks to be an alignment issue:

Current: 
[![Image from Gyazo](https://i.gyazo.com/747ecf967cd136feebf28a34f76148ff.png)](https://gyazo.com/747ecf967cd136feebf28a34f76148ff)

Desired:
[![Image from Gyazo](https://i.gyazo.com/a66e22ded081db6932dbdc67cf5f32ff.png)](https://gyazo.com/a66e22ded081db6932dbdc67cf5f32ff)

Reply:
Fixed.

[![Image from Gyazo](https://i.gyazo.com/0727ee2e0b183ac2aba906654b159d9d.png)](https://gyazo.com/0727ee2e0b183ac2aba906654b159d9d)